LanguageApp/app/screens/ProviderScenarioScreen.js

```diff
@@ -40,22 +46,59 @@ function ProviderScenarioScreen({ route }) {
     setAnswerAudio(data);
   };
 
-  // Submission data to be passed to Firebase
-  const translatedScenario = {
-    promptTranslation: cpPrompt,
-    answerTranslation: cpAnswer,
-    promptRecording: promptAudioLink,
-    answerRecording: answerAudioLink,
-    translatorID: 11111111,
-  };
+    // Submission data to be passed to Firebase
+    const translatedScenario = {
+      promptTranslation: cpPrompt,
+      answerTranslation: cpAnswer,
+      promptRecording: promptAudioLink,
+      answerRecording: answerAudioLink,
+      translatorID: 11111111,
+    };
 
   const submitTranslation = async () => {
     console.log("submitting to database", translatedScenario);
-    console.log("Prompt audio uri to submit to Firebase", promptAudio);
-    console.log("Answer audio uri to submit to Firebase", answerAudio);
-    /* TO DO:  Add code to sumbit to Firestore */
-    setCpPrompt("");
-    setCpAnswer("");
+    console.log("Prompt audio uri to submit to storage??", promptAudio);
+    console.log("Answer audio uri to submit to storage...", answerAudio);
+
+    // Create calls to use to add to DB
+    // https://firebase.google.com/docs/firestore/manage-data/add-data#update_fields_in_nested_objects
+    var answerRecordingLanguage = "answerRecording." + route.params.language;
+    var answerTranslationLanguage = "answerTranslation." + route.params.language;
+    var promptRecordingLanguage = "promptRecording." + route.params.language;
+    var promptTranslationLanguage = "promptTranslation." + route.params.language;
+    var translatorIdLanguage = "translatorId." + route.params.language;
```

Comment:
Just a note about JavaScript -- `var` is going out of use. Instead, `const` or `let` are used. Since these values don't change, they would be `const`. Otherwise, `let` would be used. You can read more about that here: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75